x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/create_job.ts

```diff
@@ -217,15 +254,18 @@ async function createADJobFromLensSavedObject(
   jobConfig.analysis_config.detectors = createDetectors(fields, splitField);
 
   jobConfig.data_description.time_field = timeField.sourceField;
-  jobConfig.analysis_config.bucket_span = DEFAULT_BUCKET_SPAN;
+  jobConfig.analysis_config.bucket_span = bucketSpan;
   if (splitField) {
     jobConfig.analysis_config.influencers = [splitField.sourceField];
   }
-
-  const createdBy =
-    splitField || jobConfig.analysis_config.detectors.length > 1
-      ? CREATED_BY_LABEL.MULTI_METRIC
-      : CREATED_BY_LABEL.SINGLE_METRIC;
+  const isSingleMetric = splitField === null && jobConfig.analysis_config.detectors.length === 1;
+  const createdBy = isSingleMetric ? CREATED_BY_LABEL.SINGLE_METRIC : CREATED_BY_LABEL.MULTI_METRIC;
```

Comment:
This all affects cloning behavior. It would be nice if it could use the single or multi-metric wizard when cloning jobs created by this path, rather than going into the advanced wizard as currently.

Reply:
the options here are to leave this as is, or to add two new labels and have the wizards know what to do with them when cloning.
I'm in favour of the latter as it would be good to track lens job creation.
So when cloning, the wizard would redirect to single metric wizard if the job is `CREATED_BY_LABEL.SINGLE_METRIC` or `CREATED_BY_LABEL.SINGLE_METRIC_FROM_LENS` (or whatever we go with)

Reply:
Updated in https://github.com/elastic/kibana/pull/136421/commits/ae35f0de4bed8502f2c0f97cacee58d0675c6f29